### PR TITLE
Helm: Change the chart to deploy 2.8.3 version of Azure IIoT platform.

### PR DESCRIPTION
**Pull request by karok2m**

### CODE REVIEW

File: docs/manual/readme.md [hunk 1]
```diff
@@ -728,7 +728,7 @@ References:
 
 [Helm](https://helm.sh/docs/) is a [Kubernetes](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/) package and operations manager, analogous to yum and apt. Application packages themselves are called Helm Charts and can be installed using the Helm tool. Helm allows users to easily templatize their Kubernetes manifests and provide a set of configuration parameters that allows users to customize their deployment.
 
-For the deployment of cloud microservices of Azure Industrial IoT platform into a Kubernetes cluster we provide **azure-industrial-iot** Helm chart. Version `0.4.2` of azure-industrial-iot Helm char should be used for deployment of `2.8.x` versions of Azure Industrial IoT microservices.
+For the deployment of cloud microservices of Azure Industrial IoT platform into a Kubernetes cluster we provide **azure-industrial-iot** Helm chart. Version `0.4.3` of azure-industrial-iot Helm char should be used for deployment of `2.8.x` versions of Azure Industrial IoT microservices.
```
Comment (marcschier) [inline docs/manual/readme.md]: Typo: "Helm char" should be "Helm chart".

File: docs/manual/readme.md [hunk 2]
```diff
@@ -2284,7 +2284,7 @@ In the below commands "`azure-industrial-iot`" is used as the namespace. Please
...
+It should be noted that different versions of Helm chart are intended for deploying different versions of the Azure Industrial IoT platform. This stems from the fact that there can be a different number of components in different versions of the Azure Industrial IoT platform, so deployment requirements and configuration parameters differ between versions. Currently, the latest version of the Helm chart is `0.4.3` which by default deploys `2.8.3` version of the components. The same version can be used for deployment of `2.8.0` and higher `2.8.x` versions. For the version compatibility matrix, please check out [how to deplyo using Helm](../deploy/howto-deploy-helm.md).
```
Comment (marcschier) [inline docs/manual/readme.md]: Typo: deplyo should be deploy

**Review by marcschier — DISMISSED** [review]
See comments on typos.

File: docs/manual/readme.md (hunk 1 above)
Comment (karok2m) [reply docs/manual/readme.md]: Fixed.

File: docs/manual/readme.md (hunk 2 above)
Comment (karok2m) [reply docs/manual/readme.md]: Fixed.

**Review by dacolgit — APPROVED** [review]

**Pull request merged**